.github/workflows/weekly-docs.yaml

```diff
@@ -33,6 +33,7 @@ jobs:
           reporter: github-pr-review
           config_file: ".github/.linkspector.yml"
           fail_on_error: "true"
+          filter_mode: "nofilter"
```

Comment:
I incorrectly assumed that the weekly-docs job being started from the cron job would mean it wouldn't use the pull request changes filter, but evidently not. This means it checks the whole `docs` folder for the cron job & on PRs.

Reply:
So on schedule, we check the whole repo?
What does the `filter_mode` mean?

Reply:
The default filter_mode was just the PR diff. Now it should be everything mentioned in the `.linkspector.yml` config.